remote-api/src/main/java/org/alfresco/repo/web/scripts/content/ContentGet.java

```diff
@@ -72,42 +74,46 @@ public class ContentGet extends StreamContent implements ServletContextAware
     private List<String> nonAttachContentTypes = Collections.emptyList();
 
     /**
-     * @param nonAttachContentTypes List<String>
+     * @param nonAttachContentTypes
+     *            List<String>
```

Comment:
Is Spotless doing this? I guess I've not seen it because we have very little Javadoc in the HxI Connector.

Reply:
Yes, this is made by Spotless